parsl/monitoring/monitoring.py

```diff
@@ -327,7 +327,7 @@ def filesystem_receiver(logdir: str, q: "queue.Queue[AddressedMonitoringMessage]
     os.makedirs(new_dir, exist_ok=True)
 
     while True:  # this loop will end on process termination
-        logger.info("Start filesystem radio receiver loop")
+        logger.debug("Start filesystem radio receiver loop")
 
```

Comment:
Without any dynamic data, this log line looks to me to be of negligible value.  Yes, the loop is iterating, but that can also be generally inferred by lack of traceback or other log messages.  What about removing it entirely, moving it out of the loop (just before the `while: True`), or adding something to it?  Perhaps the number of files the iteration would process?